okta-hosted-login/README.md

```diff
@@ -1,30 +1,65 @@
+
+### Add that same configuration to the sample's appsettings
+
+Replace the okta configuration placeholders in the `appsettings.json` with your configuration values from the [Okta Developer Console]. 
 You can see all the available configuration options in the [okta-aspnet GitHub](https://github.com/okta/okta-aspnet/blob/master/README.md).
-For step-by-step instructions, visit the Okta [ASP.NET MVC quickstart]. The quickstart will guide you through adding Okta login to your ASP.NET application.
+For step-by-step instructions, visit the Okta [Sign Users in to Your Web Application guide]. The guide will walk you through adding Okta sign-in to your ASP.NET application.
 
-Now start your server and navigate to http://localhost:8080 in your browser.
+### Run again and try to sign in
 
-If you see a home page that allows you to login, then things are working!  Clicking the **Log in** link will redirect you to the Okta hosted sign-in page.
+Click the **Sign In** link in the Home page and it will redirect you to the Okta hosted sign-in page.
 
-You can login with the same account that you created when signing up for your Developer Org, or you can use a known username and password from your Okta Directory.
+You can sign in with the same account that you created when signing up for your Developer Org, or you can use a known username and password from your Okta Directory.
 
-**Note:** If you are currently using your Developer Console, you already have a Single Sign-On (SSO) session for your Org.  You will be automatically logged into your application as the same user that is using the Developer Console.  You may want to use an incognito tab to test the flow from a blank slate.
+**Note:** If you are currently using your Developer Console, you already have a Single Sign-On (SSO) session for your Org.  You will be automatically signed into your application as the same user that is using the Developer Console.  You may want to use an incognito tab to test the flow from a blank slate.
 
-[OIDC Middleware Library]: https://github.com/okta/okta-aspnet
-[Authorization Code Flow]: https://developer.okta.com/authentication-guide/implementing-authentication/auth-code
+[Okta ASP.NET Core SDK]: https://github.com/okta/okta-aspnet
 [OIDC Web Application Setup Instructions]: https://developer.okta.com/authentication-guide/implementing-authentication/auth-code#1-setting-up-your-application
-[ASP.NET MVC quickstart]:https://developer.okta.com/quickstart/#/okta-sign-in-page/dotnet/aspnetcore
\ No newline at end of file
+[Enforce HTTPS in ASP.NET Core]: https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-2.2&tabs=visual-studio
+[Configuring HTTPS in ASP.NET Core across different platforms]:https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/
+[Sign Users in to Your Web Application guide]: https://developer.okta.com/guides/sign-into-web-app/aspnet/before-you-begin/
```

Comment:
We should link directly to `aspnetcore` here, not `aspnet`

Reply:
good catch 👍